scripts/unity/header_prepare.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+import sys
+import re
+import argparse
+
+def func_names_from_header(in_file, out_file, out_wrap_file):
```

Comment:
Should use a different function name since it clearly does something else than the other func_names_from_header